Formula/bosh-cli.rb

```diff
@@ -0,0 +1,24 @@
+class BoshCli < Formula
+  desc "Cloud Foundry BOSH CLI v2"
+  homepage "https://bosh.io/docs/cli-v2/"
+  url "https://github.com/cloudfoundry/bosh-cli/archive/v6.4.4.tar.gz"
+  sha256 "2410c243046fd9bc748257add8cee4b250e8973ffb45474e8a0ac7f83822eb97"
+  license "Apache-2.0"
+  head "https://github.com/cloudfoundry/bosh-cli.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    # https://github.com/cloudfoundry/bosh-cli/blob/master/ci/tasks/build.sh#L23-L24
+    inreplace "cmd/version.go", "[DEV BUILD]", version.to_s
```

Comment:
Since their version is [something like `$version-$commit_hash-$time`](https://github.com/cloudfoundry/bosh-cli/blob/1c35b59d37857bd913f98e4b01661ca3fe337c8e/ci/tasks/build.sh#L21), why not
```suggestion
    inreplace "cmd/version.go", "[DEV BUILD]", "#{version}-#{tap.user}-#{Time.now.utc.strftime("%Y-%m-%dT%H:%M:%SZ")}"
```
?

Reply:
nice improvement. 👍